README.md

```diff
@@ -227,39 +227,139 @@ rudderTyper.orderCompleted({
 - Execute the following command to generate a bundle from the RudderTyper client:
 
 ```sh
-browserify analytics/index.js --standalone rudderTyper >  rudderTyperBundle.js
+npx browserify analytics/index.js --standalone rudderTyper >  rudderTyperBundle.js
+```
+
+For typescript project add `tsify` npm package as dev dependency and run the following command:
+
+```sh
+npx browserify analytics/index.js -p [ tsify ] --standalone rudderTyper >  rudderTyperBundle.js
 ```
 
 - Now you can make calls from your `html` file as shown:
 
 ```html
```

Comment:
Let's not add the RudderStack snippet here. Please update the instructions to point to the specific docs section.

Reply:
Addressed